src/grid.h

```diff
@@ -608,7 +608,7 @@ struct Grid {
         const int root_grid_spacing = 2;  // Can't be adjusted in editor, so use a default.
         const int font_size = 14 - indent / 2;
         const int grid_border_width =
-            cell == doc->rootgrid ? root_grid_spacing : user_grid_outer_spacing - 1;
+            cell == doc->rootgrid.get() ? root_grid_spacing : user_grid_outer_spacing - 1;

```

Comment:
See my other comment about using it as boolean

Reply:
Also here:
I think the precedence order is 
`const int grid_border_width = ((cell == doc->rootgrid.get()) ? root_grid_spacing : user_grid_outer_spacing) - 1)`

Reply:
Unfortunately they have not, would make things easier...